src/wallet/test/wallet_tests.cpp

```diff
@@ -27,7 +27,7 @@ typedef set<pair<const CWalletTx*,unsigned int> > CoinSet;
 
 BOOST_FIXTURE_TEST_SUITE(wallet_tests, WalletTestingSetup)
 
-static CWallet wallet;
+static const CWallet wallet;
```

Comment:
Aside: We create a global static wallet object here? I never knew, assumed these tests would use the wallet created in WalletTestingSetup.